test/intl402/DateTimeFormat/prototype/formatToParts/related-year-zh.js

```diff
@@ -12,18 +13,25 @@ locale: [zh-u-ca-chinese]
 function verifyFormatParts(actual, expected, message) {
```

Comment:
it's fun how `message` was never used here.

Reply:
Lol, right, we should just remove it.